test/src/server/routes/editor.js

```diff
@@ -0,0 +1,63 @@
+import {createEditor, truncateEntities} from '../../../test-helpers/create-entities';
+
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import {getEditorActivity} from '../../../../src/server/routes/editor.js';
+import orm from '../../../bookbrainz-data';
+import {random} from 'faker';
+
+
+chai.use(chaiHttp);
+const {expect} = chai;
+
+
+describe('getEditorActivity', () => {
+	const {Revision} = orm;
+	let editorJSON;
+	before(async () => {
+		const editor = await createEditor();
+		editorJSON = editor.toJSON();
+		const revisionAttribs = {
+			authorId: editorJSON.id
+		};
+
+		const RevisionArray = [];
+		for (let i = 0; i <= 12; i++) {
+			const tempDate = new Date();
+			tempDate.setFullYear(2019, i, 1);
+			revisionAttribs.id = random.number();
+			revisionAttribs.createdAt = tempDate;
+			RevisionArray.push(
+				new Revision(revisionAttribs).save(null, {method: 'insert'})
+			);
+		}
+		await Promise.all(RevisionArray);
+	});
+	after(truncateEntities);
+
+	it('should return values for plotting graph', async () => {
+		const startDate = new Date();
+		startDate.setFullYear(2019, 1, 1);
+		const ActivityData = await getEditorActivity(editorJSON.id, startDate, Revision);
+		const MonthArray = ['Jan-19', 'Feb-19', 'Mar-19', 'Apr-19', 'May-19', 'Apr-19', 'May-19',
```

Comment:
There's an extra `'Apr-19', 'May-19'` here, which points to an issue with the test: it doesn't test what's returned.

You can create an object with exactly the values you expect to get back from the utility, then use `expect(ActivityData).to.deep.equal(myExpectedResultsObject)`.
That way, you can test at the same time both if you have the right number of results *and* if you have one per month *and* if the month string and activityData are formatted correctly.

Also, by convention, variables and functions names use camelCase notation, while starting with an uppercase is used to represent class constructors, types and a few other special cases.